delta_Go/jimmy_main.py

```diff
@@ -0,0 +1,204 @@
+import copy
```

Comment:
Should this file be in this repo? Do we want users seeing our solution code? Even if it does use stable baselines? Surely not...